### PR TITLE
add sinon-bluebird to make promises in tests a bit cleaner

### LINKED ISSUE #2115: Some tests return a reject but won't break the overall tests
# This is a "Bug Report"
## Description

If one runs `npm test` all our unit tests are run. However if you look closely at the output you can 
see some rejection printings (See screenshot below).

bildschirmfoto 2016-09-13 um 13 53 55

/cc @flomotlik @eahefnawy

### PR DESCRIPTION
**Pull request by dougmoscrop**
## What did you implement:

Closes #2115

The promise rejections were causing fake 'unhandled rejection' messages to print during the tests. As the related issue says, there's nothing wrong, but this fixes it so that the messages do not get displayed.

## How did you implement it:

Using `sinon-bluebird`.

## How can we verify it:

`npm test` - only affects unit tests.

## Todos:

- [x] Write tests
- [x] Write documentation
- [x] Fix linting errors
- [x] Make sure code coverage hasn't dropped
- [x] Provide verification config/commands/resources
- [x] Enable ["Allow edits from maintainers"](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) for this PR
- [x] Change ready for review message below

***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO

### CODE REVIEW

File: lib/plugins/package/lib/packageService.test.js [hunk 1]
```diff
@@ -3,10 +3,11 @@
 
 const expect = require('chai').expect;
 const sinon = require('sinon');
-const BbPromise = require('bluebird');
 const Package = require('../index');
 const Serverless = require('../../../../lib/Serverless');
 
+require('sinon-bluebird');
```
Comment (pmuens) [inline lib/plugins/package/lib/packageService.test.js]: Why is `sinon-bluebird` required here?

**Comment by dougmoscrop** [comment]
Thanks, yeah that was my bad.

**Comment by pmuens** [comment]
No worries! Thanks for the PR @dougmoscrop 💯

**Comment by dougmoscrop** [comment]
Fixed conflicts

**Comment by dougmoscrop** [comment]
Fixed conflicts / rebased - cc @pmuens again :D

**Comment by pmuens** [comment]
Thanks @dougmoscrop 👍 /cc @eahefnawy

**Pull request merged**